src/components/ListItems/ListItems.js

```diff
@@ -28,6 +29,46 @@ const ListItems = ({
           tabIndex={0}
           className={styles.deleteIcon}
         />
+        <FontAwesomeIcon
+          icon={faCheckCircle}
+          onClick={() => completeItem(key)}
+          role="button"
+          aria-label={`Added item: ${text}`}
+          data-testid={`complete-item-${text}`}
+          tabIndex={0}
+          className={isCompleted ? (styles.completedIcon) : (styles.completeIcon)}
+        />
+      </li>
+    ))}
+    <h3>Added items</h3>
+    {entries.filter(({ isCompleted }) => isCompleted).map(({ key, text, isCompleted }) => (
+      <li key={key} className={styles.listItem}>
+        <input
+          type="text"
+          id={key}
+          value={text}
+          onChange={(e) => updateItem(e.target.value, key)}
+          style={{ textDecoration: isCompleted ? 'line-through' : '' }}
+        />
+        {' '}
+        <FontAwesomeIcon
+          icon={faTrash}
+          onClick={() => deleteItem(key)}
+          role="button"
+          aria-label={`Delete item: ${text}`}
+          data-testid={`delete-item-${text}`}
+          tabIndex={0}
+          className={styles.deleteIcon}
+        />
+        <FontAwesomeIcon
+          icon={faCheckCircle}
+          onClick={() => completeItem(key)}
+          role="button"
+          aria-label={`Added item: ${text}`}
+          data-testid={`complete-item-${text}`}
+          tabIndex={0}
+          className={isCompleted ? (styles.completedIcon) : (styles.completeIcon)}
+        />
```

Comment:
Do you want to give moving this whole `<li>...</li>` part inside another component in this PR so it can be re-used, or would you rather do that afterwards?